### ISSUE TITLE
Parcel build fails with 5.3.0-alpha1

### ISSUE THREAD

**Issue by renspr** [twbs/bootstrap#37866]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

If I setup a Parcel project as described here https://getbootstrap.com/docs/5.3/getting-started/parcel/ the scss build fails when using bootstrap 5.3.0-alpha1.

```
🚨 Build failed.

@parcel/transformer-css: Invalid state

  /bootstrap_parcel_test/src/scss/styles.scss:2511:29
    2510 | }
  > 2511 | .form-floating::before:not(.form-control:disabled) {
  >      |                             ^
    2512 |   position: absolute;
    2513 |   top: var(--bs-border-width);
```

### Reduced test cases

Setup as described here: https://getbootstrap.com/docs/5.3/getting-started/parcel/ with 5.3.0-alpha1

### What operating system(s) are you seeing the problem on?

macOS

### What browser(s) are you seeing the problem on?

Chrome

### What version of Bootstrap are you using?

v5.3.0-alpha1

**Comment by mdo** [comment]
Confirmed. This is because of https://github.com/parcel-bundler/lightningcss/issues/149.

/cc @julien-deramond for future while he's out

**Comment by brendanheyu** [comment]
I can confirm I am also seeing this.

  "@parcel/transformer-sass": "^2.8.3",
  "parcel": "^2.8.3",
   "purgecss": "^5.0.0"
 },
  "dependencies": {
    "@popperjs/core": "^2.11.6",
    "bootstrap": "^5.3.0-alpha1"
  }

Reduced test cases
Setup as described here: https://getbootstrap.com/docs/5.3/getting-started/parcel/ with 5.3.0-alpha1

What operating system(s) are you seeing the problem on?
Windows 10

What browser(s) are you seeing the problem on?
Chrome/any

What version of Bootstrap are you using?
v5.3.0-alpha1

**Comment by renspr** [comment]
Changing `&::before:not(.form-control:disabled)` to `&:not(.form-control:disabled)::before` in https://github.com/twbs/bootstrap/blob/main/scss/forms/_floating-labels.scss#L4 seems to fix this issue.

**Comment by benpotter** [comment]
I've manually updated my _floating-labels.scss file to correct the issue as noted in @renspr comment above which fixes the issue for me as well. However when deploying to Netlify it's running into the same issue:
<img width="1195" alt="Screen Shot 2023-03-15 at 07 36 20" src="https://user-images.githubusercontent.com/4133511/225342984-4c718139-c582-4300-b45f-f7b91fc45522.png">

Isn't this fixed by changing the underlying source of Bootstrap?

Here's my package.json
`"devDependencies": {
        "@11ty/eleventy": "^2.0.0",
        "@parcel/transformer-sass": "^2.8.3",
        "autoprefixer": "^10.4.13",
        "cross-env": "^7.0.3",
        "html-minifier": "^4.0.0",
        "npm-run-all": "^4.1.5",
        "nunjucks": "^3.2.3",
        "parcel": "^2.8.3",
        "sass": "^1.58.3"
    },
    "dependencies": {
        "@popperjs/core": "^2.11.6",
        "bootstrap": "^5.3.0-alpha1",
        "bootstrap-icons": "^1.10.3"
    }`

**Comment by XhmikosR** [comment]
No, it'll be included in 5.3.0-alpha2 in the next days

**Comment by benpotter** [comment]
@XhmikosR Ok so my Netlify option right now would be to use the stable Bootstrap 5.2.3 version with Netlify? Is there another workaround or should I just wait... estimated day when 5.3.0-alpha2 will be released?
